R/simulation-batch.R

```diff
@@ -19,36 +20,74 @@ SimulationBatch <- R6::R6Class(
     initialize = function(ref, simulation) {
       validateIsOfType(simulation, Simulation)
       super$initialize(ref)
-      self$simulation <- simulation
+      private$.simulation <- simulation
     },
 
-    #' @description Set the parameter and initial values in the simulation and run the simulation
+    #' @description Add a set of parameter and start values for next execution.
+    #' @details Intended for the use with \code{runSimulationBatches}. The simulation batch is executed
+    #' with the sets of parameter and initial values that have been scheduled. The set of run values is cleared after successful run.
+    #'
     #' @param parameterValues Vector of parameter values to set in the simulation (default is `NULL`)
     #' @param initialValues Vector of initial values to set in the simulation  (default is `NULL`)
-    #' @return A `SimulationResults` object containing the result of the simulation run
-    run = function(parameterValues = NULL, initialValues = NULL) {

```

Comment:
No `run` for the simulation batch! Batches should always be run with `runSimulationBatches` function.